src/libs/actions/BankAccounts.js

```diff
@@ -216,12 +226,65 @@ function validateBankAccount(bankAccountID, validateCode) {
     });
 }
 
+/**
+ * @param {Object} data
+ * @returns {Object}
+ */
+function mergeWithLocalACHData(data) {
+    const updatedACHData = {
+        ...reimbursementAccountInSetup,
+        ...data,
+
+        // This param tells Web-Secure that this bank account is from NewDot so we can modify links back to the correct
+        // app in any communications. It also will be used to provision a customer for the Expensify card automatically
+        // once their bank account is successfully validated.
+        enableCardAfterVerified: true,
+    };
+
+    if (data && !_.isUndefined(data.isSavings)) {
+        updatedACHData.isSavings = Boolean(data.isSavings);
```

Comment:
Why do you need it? What wrong values might data.isSavings have originally?

Reply:
My guess is that we were storing `isSavings` as the string "true" or "false" instead of a Boolean. That doesn't seem to be the case anymore, so I'm removing this. Thank you!